apps/test/unit/p5lab/spritelab/SpritelabTest.js

```diff
@@ -22,6 +22,18 @@ import 'script-loader!@code-dot-org/p5.play/examples/lib/p5';
 import 'script-loader!@code-dot-org/p5.play/lib/p5.play';
 import {singleton as studioApp} from '@cdo/apps/StudioApp';
 
+const backgroundSprite = {
```

Comment:
This is probably okay because it's just a test, but it's curious that we're using an animal animation for a background rather than an actual background image.

Reply:
This was just because I knew the data for this image. Could totally update to a background.

Reply:
Okays that's what I figured! If you end up pulling the "grid" image into defaultSprites, maybe you could add that same metadata here so it's more obvious what `backgroundSprite` is meant to be.